### PR TITLE
 Package_SO: Add new SOIC 8 definition 

### PR DESCRIPTION
**Pull request by antoniovazquezblanco**
Footprint PR: https://github.com/KiCad/kicad-footprints/pull/1601

### CODE REVIEW

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml [hunk 1]
```diff
@@ -350,6 +349,27 @@ SOIC-8-1EP_3.9x4.9mm_P1.27mm_EP2.514x3.2mm:
     # bottom_pad_size:
     paste_avoid_via: False
 
+SOIC-8_5.275x5.275mm_P1.27mm:
+  size_source: 'http://ww1.microchip.com/downloads/en/DeviceDoc/20005045C.pdf'
```
Comment (poeschlr) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml]: add #page=23 to point directly to the correct page

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml [hunk 2]
```diff
@@ -350,6 +349,27 @@ SOIC-8-1EP_3.9x4.9mm_P1.27mm_EP2.514x3.2mm:
...
+  body_size_x:
+    minimum: 5.15
+    maximum: 5.4
+  body_size_y:
+    minimum: 5.15
+    maximum: 5.4
```
Comment (poeschlr) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml]: please add the overall_height parameter to allow reuse of this info for 3d model generation

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml (hunk 1 above)
Comment (antoniovazquezblanco) [reply scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml]: Done

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml (hunk 2 above)
Comment (antoniovazquezblanco) [reply scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml]: Done

**Comment by poeschlr** [comment]
@pointhi the footprint has been merged this is good to go.

**Comment by antoniovazquezblanco** [comment]
Thank you all!

**Pull request merged**